tools/gce-usage-log/README.md

```diff
@@ -182,10 +197,116 @@ bq mk \
+* `your_interval_amount` is any integer representing the frequency of intervals to calculate. 
+For example, if you wanted to create a time-series dataset looking at the inventory of VMs on an 
+hourly cadence, you would choose `HOUR` for `TIME_INTERVAL_UNIT` and `1` for `TIME_INTERVAL_AMOUNT`.
+If you wanted to see the inventory at 5-minute increments, you would choose `MINUTE` and 5, respectively.
+
+
+Next, create a reference to the interval view:
+
+```
+export INTERVAL_VIEW=$(cat gce_interval_view.sql | sed -e "s/_PROJECT_/$PROJECT_ID/g" -e "s/_TIME_INTERVAL_UNIT_/$TIME_INTERVAL_UNIT/g" -e "s/_TIME_INTERVAL_AMOUNT_/$TIME_INTERVAL_AMOUNT/g")
+```
+
+Now, create the view.
+````
+bq query \
+--project $PROJECT_ID \
+--use_legacy_sql=false \
+--destination_table=gce_usage_log.$DESTINATION_TABLE \
+--display_name="Interval usage of GCE Usage Logs" \
+--replace=true \
+--schedule='every 24 hours' $INTERVAL_VIEW
+````
+Note: As a default, it is configured to run every 24 hours, but you can scheduled it more/less frequently as needed.
+There is also no default expiration set, but this can be added if you only need historical data from a certain timeframe.
+
+
 ## 3. Using the dataset
 
+### 3.2.1 How to Use the Interval View
+
 Now that your dataset is ready, how do you query it?
 
+The interval view allows to see aggregated point-in-time statistics. To find resource usage for a specific time-frame, you can query for the aggregate usage information for the month of 
+September by using the sample query below as an example. You can adjust the timeframe by altering the WHERE clause
+or choose to only select certain fields depending on what you are trying to predict.
+
+
+```bash
+SELECT
+  custom_interval as hour,
+  count(instance_id) as num_instances,
+  SUM(cores) as total_cores,
+  SUM(memory_mb) as total_memory_mb,
+  SUM(pd_standard_size_gb) as total_pd_standard_size_gb,
+  SUM(pd_ssd_size_gb) as total_pd_ssd_size_gb,
+  SUM(local_ssd_size_gb) as total_local_ssd_size_gb
+  
+FROM `gce_usage_log._gce_usage_log_interval`
+
+WHERE
+  custom_interval >= "2019-09-01" AND custom_interval < "2019-10-01"
+
+GROUP BY 1
+```
+
+The results will look something like this (Note that the aggregated statistics will most likely 
+vary for workloads as VM resources change over hour, but they do not in this example.)
+
+![interval_query](images/interval-query-results.png)
+
+### 3.2.2 How to Create a Time-Series Graph on the Interval View
+
+If you want to see the same data in a time-series graph rather than a data table, you can do this 
+in Data Studio. This allows you to create time-series graphs to monitor changes and spikes of inventory 
+over time. This can be done on whichever metrics that your team would like to use for capacity planning, 
+such as looking at the total count of instances over time, cores, memory, etc.
+
+1. To create a time-series graph, open up [Data Studio](https://www.datastudio.google.com).
```

Comment:
could this be simplified by having a link to an existing public dashboard where user only has to change the data source ?
Similar to [this asset](https://github.com/GoogleCloudPlatform/professional-services/tree/master/examples/bigquery-audit-log#3-copying-the-data-source-in-data-studio)

Reply:
Looking into this, thanks!